### ISSUE TITLE
Plugin is not working with sails v1.x

### ISSUE THREAD

**Issue by OPADA-Eng** [kristian-ackar/sails-hook-blueprint-count#5]
The plugin is not working any more with the current version of sails js 1.0
can you please post an upgrade soon ?

**Comment by kristian-ackar** [comment]
I would appreciate if you create a pull request with patch :-)

**Comment by pixnlove** [comment]
FYI, in SailsJS v1.0, `sails-hook-blueprint-count` is natively integrated.
https://sailsjs.com/documentation/reference/waterline-orm/models/count

**Comment by dehimer** [comment]
> FYI, in SailsJS v1.0, `sails-hook-blueprint-count` is natively integrated.
> https://sailsjs.com/documentation/reference/waterline-orm/models/count

For controllers? For models it is was available before.

**Comment by pixnlove** [comment]
@dehimer yes you can call it with `await Model.count(criteria)` in your controller now.

**Comment by dehimer** [comment]
> @dehimer yes you can call it with `await Model.count(criteria)` in your controller now.

I implemented count now. Added count action to base/Controller and added routes to config/routes.
If you will not specify route - you will get undefined model in options and get error in 
const Model = actionUtil.parseModel(request);

**Comment by dehimer** [comment]
```
module.exports = {
  /*
   * Generic count action for controller.
   *
   * @param   {Request}   request
   * @param   {Response}  response
   */
  count(request, response) {
    console.log('count');
    const Model = actionUtil.parseModel(request);
    const criteria = actionUtil.parseCriteria(request);

    Model
      .count(criteria)
      .exec((error, count) => {
        if (error) {
          console.log(error);
          response.negotiate(error);
        } else {
          console.log(count);
          response.ok({ count });
        }
      });
  },
};
```

**Comment by Lorex** [comment]
@dehimer How do I specify my routes?

I tried the following code but get "No model specified in route options." error 😢 

`
  'GET /*/count' : {
    action: 'main/count'
  }`

**Comment by dehimer** [comment]
> @dehimer How do I specify my routes?
> 
> I tried the following code but get "No model specified in route options." error 😢
> 
> ` 'GET /*/count' : { action: 'main/count' }`

I did it in this way:
```
'GET /timeentry/count': {
    controller: 'TimeEntryController',
    action: 'count',
    model: 'timeentry'
  },
```
For every model.
Looks ugly, but works.
